client/src/components/form/input-component.tsx

```diff
@@ -0,0 +1,217 @@
+          <span className="max-w-full truncate">
+            <SelectValue placeholder={placeholder || 'Select'} />
+          </span>
+        </SelectTrigger>
+        <SelectContent className="max-w-[632px]">
+          {options?.map(({ label, value }) => (
+            <SelectItem key={value} value={value}>
+              {label}
+            </SelectItem>
+          ))}
+        </SelectContent>
+      </Select>
+    );
+  }
+  if (type === 'textarea') {
+    const watchField = watch(name) as string;
+    const counterId = `${name} - counter`;
+    const hasError: boolean = !!watchField && !!maxSize && watchField.length > maxSize;
+    return (
+      <>
+        <Textarea
+          {...field}
+          value={value ?? ''}
+          required={required}
+          error={hasError}
+          className={cn({ 'min-h-[172px]': maxSize && maxSize > 350 })}
+          id={id}
+          aria-describedby={ariaDescribedBy ? `${ariaDescribedBy} ${counterId}` : counterId}
+          aria-invalid={!!ariaInvalid}
+        />
+        {maxSize && (
+          <div
+            id={counterId}
+            className={cn('flex justify-end text-sm text-gray-500', {
+              'text-destructive': hasError,
+            })}
+          >
+            {watchField ? watchField.length : '0'} / {maxSize}
+          </div>
+        )}
+      </>
+    );
+  }
+  if (type === 'multiselect') {
+    return (
+      <MultiCombobox
+        id={id}
+        name={label}
+        value={(value as string[]) ?? []}
+        ariaDescribedBy={ariaDescribedBy}
+        ariaInvalid={!!ariaInvalid}
+        onChange={onChange}
+        variant={variant}
+        options={options ?? []}
+        showSelected
+      />
+    );
+  }
+  if (type === 'date') {
+    const startDate = form.watch('start_date');
+    const endDate = form.watch('end_date');
+    return (
+      <Popover>
+        <PopoverTrigger asChild>
+          <Button
+            variant="vanilla"
+            size="auto"
+            className={cn(
+              'relative w-full justify-start border border-gray-300 p-4 pr-12 text-base focus-visible:!outline-1 focus-visible:!outline-offset-0 focus-visible:!outline-gray-300 focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-[3px] data-[state=open]:border-gray-400',
+              { 'border-destructive': ariaInvalid },
+            )}
+          >
+            <span className="flex items-center gap-2">
+              <CalendarIcon />
+              {value ? (
+                format({ id: 'formatDate', value })
+              ) : (
+                <span className="text-gray-500">Select date</span>
+              )}
+            </span>
+            <ChevronDown className="absolute right-4 top-4 h-6 w-6 flex-shrink-0" />
+          </Button>
+        </PopoverTrigger>
+        <PopoverContent
+          className="w-[330px] overflow-y-auto rounded-none p-0 text-base shadow-md"
```

Comment:
On the [design](https://www.figma.com/file/aMDHzK89eiqhpdKtpcoUGe/Design-system---ORCaSa?type=design&node-id=1936-3838&mode=design&t=VUS25IV2AF6p0JiM-0), there is only a shadow, no border. So what you implemented is correct.

That said, all the other dropdowns of the forms have a border (in addition to the shadow) so I wonder if we shouldn't add one as well.